src/DataSource/Http.hs

```diff
@@ -145,7 +177,7 @@ postTicketComment ZendeskResponse{..} = do
     let req2 = addJsonBody
                    (Ticket
                        (Comment (CommentId 0) (CommentBody $ "**Log classifier**\n\n" <> zrComment) [] zrIsPublic (cfgAgentId cfg))
-                       (cfgAssignTo cfg)
+                       Nothing
```

Comment:
What if an _assignee_ is already selected? This will just write down a comment with no "owner"?

Reply:
I've tested it, the classifier won't touch the assignee field at all. It just leave the field as it is.

Reply:
Awesome, thanks for the confirmation.

Reply:
Yea I think I'll add comment about them as well